packages/astroport_periphery/src/lockdrop.rs

```diff
@@ -326,3 +330,22 @@ pub struct PendingAssetRewardResponse {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct MigrateMsg {}
+
+pub fn query_proxy_reward_token(
```

Comment:
Where do we use this function?

Reply:
Removed it as it was used only in migration.